.changeset/funny-hotels-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-pwa/composables-next": minor
+---
+
+Added `getCategoryBreadcrumbs` method for fetching backend breadcrumbs in `useBreadcrumbs` composable
```

Comment:
the `buildDynamicBreadcrumbs` is more important functionality, should be mentioned and explained here more